### PR TITLE
Added simple Maps api into webserver.

### PR DESCRIPTION
**Pull request by aannie-y**
Implemented createMap() function in script.js. Index.html calls this function on-load and added simple styling in style.css.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -84,4 +84,10 @@ function deleteComment(comment) {
...
   params.append('id', comment.id);
   fetch('/delete-data', {method: 'POST', body: params});
-}
\ No newline at end of file
+}
+
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: -33.865, lng: 151.209}, zoom: 12});
```
Comment (jupi3200) [inline portfolio/src/main/webapp/script.js]: Can we have a constant explaining the location?

Comment (aannie-y) [reply portfolio/src/main/webapp/script.js]: yes will change that

Comment (aannie-y) [reply portfolio/src/main/webapp/script.js]: Done!

File: portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 /**
  * Handles upload request and gets image url.
  */
-@WebServlet("/blobstore-upload-url")
```
Comment (lilian-hunt) [inline portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java:28]: How is this related to the map api?

Comment (aannie-y) [reply portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java:28]: It's not, but for a while when I had a bug with my blobstore, this was the problem causing it so when i changed it, it messed up with all my PRs as I didn't change it in its specific branch. So this appears in all my PRs now :(

**Review by KathleenQ — COMMENTED** [review]
LGTM

**Review by jupi3200 — APPROVED** [review]
